src/App.js

```diff
@@ -1,25 +1,5 @@
-import logo from './logo.svg';
-import './App.css';
-
 function App() {
```

Comment:
maybe you could use a new syntax - arrow function?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions 
https://medium.com/codingthesmartway-com-blog/modern-react-from-the-beginning-ep5-arrow-functions-and-object-destructuring-468e5ceb6c77

Reply:
I changed that. thanks for link